openfisca_core/legislation.xsd

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+    <xs:element name="NODE" type="nodeType"/>
+
+    <xs:group name="piecewiseConstant">
```

Comment:
All numeric parameters in the XML are piecewise constant functions of the time (or step functions), cf https://en.wikipedia.org/wiki/Step_function. Maybe "stepFunction" would be clearer ?